dropwizard-testing/src/test/java/io/dropwizard/testing/junit/DropwizardAppRuleWithExplicitTest.java

```diff
@@ -0,0 +1,72 @@
+package io.dropwizard.testing.junit;
+
+import io.dropwizard.Application;
+import io.dropwizard.jetty.ConnectorFactory;
+import io.dropwizard.jetty.HttpConnectorFactory;
+import io.dropwizard.server.DefaultServerFactory;
+import io.dropwizard.setup.Environment;
+
+import java.util.Map;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableMap;
+
+public class DropwizardAppRuleWithExplicitTest {
+
+    @Test
+    public void bogusTest() { }
```

Comment:
Right. So, basically it exercises the new method, passing `config`, instead of a path to YAML resource from which `Configuration` would be read. Most of the stuff is fluff, scaffolding, the meat is just the construction of `DropwizardAppRule` and verifying client returns expected message.